src/modules/application/components/HomePage/HomePage.jsx

```diff
@@ -5,11 +5,13 @@ import ReportsList from 'modules/reports';
 import { actions, selectors as reportsSelectors } from 'modules/entities/modules/reports';
 import { useDispatch, useSelector } from 'react-redux';
 import * as felaRules from './HomePage.styles';
-import FilterBar from 'modules/ui/components/FilterBar';
+import FilterBar from 'modules/filters/components/FilterBar';
+import FilterFetcher from 'modules/filters/components/FilterFetcher';
```

Comment:
Those two lines should be merged in one line 

import { FilterBar, FilterFetcher } from 'modules/filters/components/FilterBar';

Reply:
Done in #10